src/components/Differential/Differential.jsx

```diff
@@ -1980,8 +1980,8 @@ class Differential extends Component {
           </Grid.Column>
           <Grid.Column
             className="DifferentialContentContainer"
-            mobile={16}
-            tablet={16}
+            mobile={12}
```

Comment:
This mobile/table size change is all this PR does (the rest is just changes from Prettier auto-formatting).  "mobile" is the size that is relevant when a user is zoomed to 150% on a regular sized laptop